File: fotocopiadora.erl
-module(boleta).
-export([main/0]).
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
main() ->
    io:format("ingresa la cantidad de hojas por favor")
    {ok, [Hojas]} = io:fread("", "~d"),
    PrecioPorHoja = 20
    TotalBase = hojas * PrecioPorHoja
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
    Total = if hojas > 30 -> TotalBase * 0.90;
               true -> TotalBase
            end,
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%               
    io:format("presiona enter para continuar por favor")
    io:get_line("")
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
    io:format("~c[H~c[J", [27, 27]),
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
    io:format("
********Boleta******
cantidad de hojas:  ~p
valor por hoja $ ~p
TOTAL: $~.2f~n", [hojas, PrecioPorHoja, Total]).
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%            

File: fotocopiadora.scala
import scala.io.StdIn
////////////////////////////////
object fotocopiadora {
    def main(args: Array [String]): Unit = {
        if(System.getProperty("os.name").toLowerCase().contains("windows")) {
           new ProcessBuilder("cmd", "/c", "cls").inheritIO().start().waitFor() 
         } else {
            new ProcessBuilder("clear").inheritIO().start.waitFor()
         }
///////////////////////////////////
            print("ingresa la cantidad de hojas por favor")
            val hojas = StdIn.readInt()
///////////////////////////////////
            val precioPorHoja = 20
////////////////////////////////////
            var total = precioPorHoja * hojas
/////////////////////////////////////
            if (hojas > 30) {
                total = (total * 0.90).toInt
            }
//////////////////////////////////////
            println("presiona enter para continuar")
            StdIn.readLine()
////////////////////////////////////////
            if (System.getProperty("os.name").toLowerCase().contains("windows")) { 
                new ProcessBuilder("cmd", "/c", "cls").inheritIO().start().waitFor()
            }  else {
                new ProcessBuilder("clear").inheritIO().start().waitFor()
            }
/////////////////////////////////////////

            println(
                s"""""
                *******boleta******
                cantidad de hojas $hojas
                valor por hoja $$20
                total $$total
                """
        )
    }             
}
////////////////////////////////////////////
